yt_idefix/fields.py

```diff
@@ -47,3 +50,247 @@ def setup_fluid_fields(self):
         setup_magnetic_field_aliases(
             self, "idefix-dmp", [f"Vc-BX{idir}" for idir in "123"]
         )
+
+
+class PlutoXdmfFields(FieldInfoContainer):
+    _pres_units = "code_pressure"
+    _erg_units = "code_mass * (code_length/code_time)**2"
+    _rho_units = "code_mass / code_length**3"
+    _mom_units = "code_mass / code_length**2 / code_time"
+    _vel_units = "code_length/code_time"
+
+    known_other_fields = (
+        # Each entry here is of the form
+        # ( "name", ("units", ["fields", "to", "alias"], # "display_name")),
+        ("rho", (_rho_units, ["density", "rho"], None)),
+        ("vx1", (_vel_units, ["vel_x"], None)),
+        ("vx2", (_vel_units, ["vel_y"], None)),
+        ("vx3", (_vel_units, ["vel_z"], None)),
+        ("prs", (_pres_units, ["prs", "pres", "pressure"], None)),
+    )
+
+    known_particle_fields = ()
+
+    def setup_fluid_fields(self):
+        unit_system = self.ds.unit_system
+
+        # Add tracer fields
+        for i in range(1, self.ds.ntracers + 1):
+            if (self.ds._dataset_type, "tr%d" % i) in self.field_list:
```

Comment:
please avoid `%`-style formatted strings and prefer fstrings

Reply:
Yes I'll put f-strings.